packages/tracing/test/browser/request.test.ts

```diff
@@ -47,8 +47,8 @@ describe('instrumentOutgoingRequests', () => {
 describe('callbacks', () => {
   let hub: Hub;
   let transaction: Transaction;
-  const alwaysCreateSpan = () => true;
-  const neverCreateSpan = () => false;
+  const always = () => true;
+  const never = () => false;
```

Comment:
```suggestion
  const alwaysCreateSpan = () => true;
  const neverCreateSpan = () => false;
  const alwaysAttachHeaders = () => true;
  const neverAttachHeaders = () => false;
```

I get that to a certain extent this is redundant, but then it allows your tests below to look like

```js
fetchCallback(fetchHandlerData, alwaysCreateSpan, neverAttachHeaders, spans)
```

rather than

```js
fetchCallback(fetchHandlerData, always, never, spans)
```

which is significantly more self-documenting.